experimental/saitypesextensions.h

```diff
@@ -42,6 +42,42 @@ typedef enum _sai_object_type_extensions_t
 
     SAI_OBJECT_TYPE_TABLE_META_TUNNEL_ENTRY,
 
+    SAI_OBJECT_TYPE_DIRECTION_LOOKUP_ENTRY,
+
+    SAI_OBJECT_TYPE_SLB_DECAP_ENTRY,
+
+    SAI_OBJECT_TYPE_ENI_LOOKUP_TO_VM_ENTRY,
+
+    SAI_OBJECT_TYPE_INBOUND_ROUTING_ENTRY,
+
+    SAI_OBJECT_TYPE_PA_VALIDATION_ENTRY,
```

Comment:
according to the p4, there is a validation for the PA for inbound.
currently our "PA" is determined by "ip link/ifconfig of the ports" before the application starts, and the application only reads this value upon init. do you mean we need to set it from within the application and not externally?
please check if the way we did it is acceptable, because if not and need to change on the fly, it will be large effort to do.